### PR TITLE
Clarify Tippy CSS as dependency for collapsible sidebar

### PR DESCRIPTION
**Pull request by ralphjsmit**
Proposal for adding Tippy CSS dependency to the appearance page. 

Have a good weekend!

### CODE REVIEW

File: packages/admin/docs/07-appearance.md [hunk 1]
```diff
@@ -65,6 +65,21 @@ In `config/filament.php`, set the `layouts.sidebar.is_collapsible_on_desktop` to
 ],
 ```
 
+If you enable the collapsible sidebar, make sure to load the Tippy CSS-files as well, by adding the following to your service provider:
+
+
+```php
+Filament::registerStyles([
+    'https://unpkg.com/tippy.js@6/dist/tippy.css',
```
Comment (pxlrbt) [inline packages/admin/docs/07-appearance.md:73]:
```suggestion
    'https://unpkg.com/tippy.js@6/dist/tippy.css',
    'https://unpkg.com/tippy.js@6/themes/light.css',
```

File: packages/admin/docs/07-appearance.md [hunk 2]
```diff
@@ -65,6 +65,21 @@ In `config/filament.php`, set the `layouts.sidebar.is_collapsible_on_desktop` to
...
+]);
+```
+
+If you're building your own theme (see below), you can alternatively compile the Tippy CSS directly into your own CSS-file:
+
+```css
+@import 'https://unpkg.com/tippy.js@6/dist/tippy.css';
```
Comment (pxlrbt) [inline packages/admin/docs/07-appearance.md:80]:
```suggestion
@import 'https://unpkg.com/tippy.js@6/dist/tippy.css';
@import 'https://unpkg.com/tippy.js@6/themes/light.css';
```

**Review by pxlrbt — CHANGES_REQUESTED** [review]
For dark mode there's also the light theme required.

**Comment by ralphjsmit** [comment]
> For dark mode there's also the light theme required.

Committed your suggested changes, thanks!

**Comment by ryangjchandler** [comment]
Should we not be compiling the Tippy CSS into the admin CSS?

**Comment by pxlrbt** [comment]
We do. This is for people using their own theme.
My suggestion would be to separate general theme and vendor themes.

**Comment by ryangjchandler** [comment]
Ah, my bad. Only saw feed from emails!

File: packages/admin/docs/07-appearance.md [hunk 3]
```diff
@@ -65,6 +65,23 @@ In `config/filament.php`, set the `layouts.sidebar.is_collapsible_on_desktop` to
 ],
 ```
 
+If you enable the collapsible sidebar, make sure to load the Tippy CSS-files as well, by adding the following to your service provider:
```
Comment (danharrin) [inline packages/admin/docs/07-appearance.md:68]: If it's installed by default, why do we need this?

Comment (pxlrbt) [reply packages/admin/docs/07-appearance.md:68]: We need this when people create their own theme.

Comment (danharrin) [reply packages/admin/docs/07-appearance.md:68]: I meant the first clause about using `Filament::register`, before theming is mentioned?

Comment (pxlrbt) [reply packages/admin/docs/07-appearance.md:68]: Oh sure. I think that whole part belongs in the "Building themes" section.

**Pull request merged**